zerver/lib/test_fixtures.py

```diff
@@ -296,3 +296,15 @@ def destroy_leaked_test_databases(expiry_time: int = 60 * 60) -> int:
     if p.returncode != 0:
         raise RuntimeError("Error cleaning up test databases!")
     return len(databases_to_drop)
+
+def remove_test_run_directories(expiry_time: int = 60 * 60) -> int:
+    removed = 0
+    directories = glob.glob(os.path.join(UUID_VAR_DIR, "test-backend", "run_*"))
+    for test_run in directories:
+        if round(time.time()) - os.path.getmtime(test_run) > expiry_time:
+            try:
+                os.removedirs(test_run)
```

Comment:
why `os.removedirs` for this?

Reply:
These errors in the last commit were careless, sorry about that.